app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -330,7 +335,8 @@ def update
       else
         return render status: :unprocessable_entity, json: { error: "Invalid parameter to be changed" }
       end
-    else
+    # elsif group_type == UserGroup.group_types[:banned_competitors]
+    # else
```

Comment:
Why is there commented-out (dead) code here?

Reply:
I'll add them once https://github.com/thewca/worldcubeassociation.org/pull/9395 is merged since this PR is dependent on that 😬

Reply:
Now that's merged, yay!